techniques/system/common/1.0/cf-served.st

```diff
@@ -246,8 +246,9 @@ body server control
         # last single quote in cfruncommand is left open, so that
         # arguments (like -K and --remote-bundles) are properly appended.
         cfruncommand => "${def.cf_runagent_shell} -c \'
-                            ${sys.cf_agent} -I -D cfruncommand -f failsafe.cf  \&\&
-                            ${sys.cf_agent} -I -D cfruncommand";
+                            ${g.rudder_command} agent update &&
```

Comment:
and you will need an access rule for g.rudder_command, like
`"$(g.rudder_command)"
      comment => "Grant access to rudder agent command for cfruncommand",
      admit => { ${def.policy_server} };`